packages/vue/src/router.ts

```diff
@@ -74,15 +91,12 @@ export function vueRouterInstrumentation(router: VueRouter): VueRouterInstrument
       }
 
       if (startTransactionOnPageLoad && isPageLoadNavigation) {
-        startTransaction({
-          name: transactionName,
-          op: 'pageload',
-          tags,
-          data,
-          metadata: {
-            source: transactionSource,
-          },
-        });
+        const pageloadTransaction = getActiveTransaction();
+        if (pageloadTransaction) {
+          pageloadTransaction.setName(transactionName, transactionSource);
+          pageloadTransaction.setData('params', data.params);
+          pageloadTransaction.setData('query', data.query);
```

Comment:
Made a TSC topic here: https://www.notion.so/sentry/Next-Up-76f43b476f6b4917a73bf3b37414ff41#2c2464a5529d441c8d4ef3a1fc8c8283

Reply:
Interesting. I propose we move forward with this for the time being, since `data` was already populated in the same way before and we make a follow-up PR afterwards.

Reply:
yup sounds good to me!